front-end/src/components/Register.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+import axios from "axios";
+import validate from "./handleErrorRegister";
+
+const Register = (props) => {
+  const [values, setValues] = useState({
+    username: "",
+    email: "",
+    password: "",
+    password2: "",
+    phone: "",
+  });
+
+  const [errors, setErrors] = useState({});
+
+  const handleChange = (e) => {
+    console.log(e);
+    setValues({
+      ...values,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = (e) => {
+    // console.log("Values", values);
+
```

Comment:
remove the commented out line and the empty line

Reply:
Done